skopt/optimizer/base.py

```diff
@@ -22,7 +21,7 @@ def base_minimize(func, dimensions, base_estimator,
                   acq_func="EI", acq_optimizer="lbfgs",
                   x0=None, y0=None, random_state=None, verbose=False,
                   callback=None, n_points=10000, n_restarts_optimizer=5,
-                  xi=0.01, kappa=1.96, n_jobs=1):
+                  xi=0.01, kappa=1.96, n_jobs=1, args=None):
```

Comment:
Even though the scipy optimize API uses args, it might be better to use `func_kwargs` as a dict  to conveniently pass them into the black-box function such that the order does not matter and API consistency when we refactor into `optimizer_kwargs`, `acq_func_kwargs` and such.

Reply:
So leave `gp_minimize` and co using `args=()` but switch `base_minimize` to pass it on as a dictionary?

We had discussed keyword args or not at the sprint and decided that following the `scipy.optmize` API would be better. I don't have a strong opinion beyond keeping the API as consistent as possible.

Reply:
sounds ok. @Anaaa are you still there? Can you please add a test?